File: length-of-last-words/LengthOfLastWords.cs
using System;

public class LengthOfLastWords {
    public int LengthOfLastWord(string word) {
        int end = word.Length - 1;
        
        while(end >= 0 && Char.IsWhiteSpace(word[end])) {
            end--;
        }

        int left = end;
        while(left >= 0 && !Char.IsWhiteSpace(word[left])) {
            left--;
        }

        return end - left;
    }
}

File: length-of-last-words/LengthOfLastWords.java
import java.util.*;

public class LengthOfLastWords {
    public int lengthOfLastWord1(String word) { 
        int end = word.length() - 1;
        
        while(end >= 0 && Character.isWhitespace(word.charAt(end))) {
            end -= 1;
        }

        int left = end;
        while(left >= 1 && !Character.isWhitespace(word.charAt(left))) {
            left -= 1;
        }

        return end - left;
    }
}
